code/datums/emotes.dm

```diff
@@ -0,0 +1,165 @@
+#define EMOTE_VISIBLE 1
+#define EMOTE_AUDIBLE 2
+
+/datum/emote
+	var/key = "" //What calls the emote
+	var/key_third_person = "" //This will also call the emote
+	var/message = "" //Message displayed when emote is used
+	var/message_param = "" //Message to display if a param was given
+	var/emote_type = EMOTE_VISIBLE //Whether the emote is visible or audible
+	var/restraint_check = FALSE //Checks if the mob is restrained before performing the emote
+	var/muzzle_ignore = FALSE //Will only work if the emote is EMOTE_AUDIBLE
+	var/muzzled_noise = "" // For snowflaking the muzzled noise
+	var/list/mob_type_blacklist_typecache //Types that are NOT allowed to use that emote
+	var/list/mob_type_ignore_stat_typecache
+	var/punct = "."
+	var/stat_allowed = CONSCIOUS
+	var/cooldown = 0
+	var/sound
+	var/sound_volume = 50
+	var/sound_vary = 0
+	var/sound_frequency = 0
+	var/static/list/emote_lists = list(
```

Comment:
This is a big nope no holy fucking god no. Static lists being used in this way is a horrifying mess that you'd expect to see coming out of a language like Java.

Reply:
I only did this cause of the post I linked by MSO that said static lists were more efficient for compilation than global lists. I could change it to a global list system tho